quil-rs/src/instruction/classical.rs

```diff
@@ -99,6 +107,8 @@ impl Quil for ArithmeticOperator {
 }
 
 #[derive(Clone, Debug, Hash, PartialEq, Eq)]
+#[pyclass(module = "quil.instructions", eq, frozen, hash, get_all)]
+#[pyo3(rename_all = "snake_case")]
 pub enum BinaryOperand {
     LiteralInteger(i64),
```

Comment:
Should this be `"SCREAMING_SNAKE_CASE"`? I'm less certain than the other comment on this because these variants have associated data